templates/RELEASE_NOTES_unified.htm.tmpl

```diff
@@ -18,6 +18,45 @@
     </ul>
     {{- end }}
 
+    <!--
+      This section should be in a partial on its own but we can't do that until issue
+      https://github.com/cyberark/conjur-oss-helm-chart/issues/50 is done
+    -->
+    <h2>Installation Instructions for the Suite Release Version of Conjur</h2>
+    {{- $conjurVersion := .ComponentReleaseVersion "cyberark/conjur" -}}
```

Comment:
I think it might be good practice to move local variable declarations in templates to the top of the file.

Reply:
I totally agree but I wanted to keep the logic that should be extracted into a template together. I'll make the change though.